static/js/admin.js

```diff
@@ -321,25 +324,11 @@ function _setup_page() {
         language_list: page_params.language_list,
         realm_default_language: page_params.realm_default_language,
         realm_waiting_period_threshold: page_params.realm_waiting_period_threshold,
+        is_admin: page_params.is_admin,
     };
 
     var admin_tab = templates.render('admin_tab', options);
     $("#settings_content .administration-box").html(admin_tab);
-    $("#administration-status").expectOne().hide();
-    $("#admin-realm-name-status").expectOne().hide();
-    $("#admin-realm-restricted-to-domain-status").expectOne().hide();
-    $("#admin-realm-invite-required-status").expectOne().hide();
-    $("#admin-realm-invite-by-admins-only-status").expectOne().hide();
-    $("#admin-realm-authentication-methods-status").expectOne().hide();
-    $("#admin-realm-create-stream-by-admins-only-status").expectOne().hide();
-    $("#admin-realm-add-emoji-by-admins-only-status").expectOne().hide();
-    $("#admin-realm-message-editing-status").expectOne().hide();
-    $("#admin-realm-default-language-status").expectOne().hide();
-    $('#admin-realm-waiting_period_threshold_status').expectOne().hide();
-    $("#admin-emoji-status").expectOne().hide();
-    $('#admin-filter-status').expectOne().hide();
-    $('#admin-filter-pattern-status').expectOne().hide();
-    $('#admin-filter-format-status').expectOne().hide();
```

Comment:
So this refactoring should be a separate commit.  But also, have you tested this procedure:
(1) Do something in the settings UI that triggers a "changes successful" alert
(2) Close settings
(3) Reopen settings
to see if the alert is still there?

If so, we might want to just make this more of a wildcard `$(#settings-overlay .alert).hide()` kind of thing.